storio-sqlite/src/main/java/com/pushtorefresh/storio3/sqlite/impl/DefaultStorIOSQLite.java

```diff
@@ -424,9 +427,9 @@ public Cursor query(@NonNull Query query) {
         public long insert(@NonNull InsertQuery insertQuery, @NonNull ContentValues contentValues) {
             return sqLiteOpenHelper
                     .getWritableDatabase()
-                    .insertOrThrow(
+                    .insert(
                             insertQuery.table(),
-                            insertQuery.nullColumnHack(),
```

Comment:
As SupportSQLiteDatabase doesn't allow nullColumnHack, and we should remove it from InsertQuery too